oas

```diff
@@ -1,506 +1,354 @@
-        "description": "Allowed time window sizes. A time window is the distance in time between one data point and the next (possibly null valued) in the time series. ISO8601 encoded.",
-        "type": "string",
-        "format": "duration",
-        "example": "P1H"
-      },
-      "Boundary": {
-        "description": "A geographical area expressed as a GeoJSON of type polygon to filter which sources of data must participate in the response to the query. See https://json.schemastore.org/geojson.json",
-        "type": "object",
-        "required": [
-          "type",
-          "coordinates"
-        ],
-        "properties": {
-          "type": {
-            "pattern": "^Polygon$",
-            "type": "string"
-          },
-          "coordinates": {
-            "description": "See https://json.schemastore.org/geojson.json"
-          }
-        },
-        "example": {
-          "type": "Polygon",
-          "coordinates": [
-            [
-              [
-                11.029761135578156,
-                49.57058013297548
-              ],
-              [
-                11.030300259590149,
-                49.57058013297548
-              ],
-              [
-                11.030300259590149,
-                49.57087931601995
-              ],
-              [
-                11.029761135578156,
-                49.57087931601995
-              ],
-              [
-                11.029761135578156,
-                49.57058013297548
-              ]
-            ]
-          ]
-        }
-      },
-      "Error": {
-        "description": "This defines the structure of an error payload when one is present. Actual messages and additional error information content may change from one implementation to another",
-        "type": "object",
-        "required": [
-          "message"
-        ],
-        "properties": {
-          "message": {
-            "type": "string",
-            "description": "Error description.",
-            "example": "General description"
-          },
-          "errors": {
-            "description": "Additional detailed information",
-            "type": "array",
-            "items": {
-              "type": "string"
+            "AssetDataResponse": {
+                "title": "AssetDataResponse",
+                "type": "object",
+                "properties": {
+                    "identifier": {
+                        "type": "string",
+                        "description": "Physical identifier (MAC Address) of the returnable asset",
+                        "example": "12:34:56:78:90:AB"
+                    },
+                    "data": {
+                        "type": "array",
+                        "items": {
+                            "$ref": "#/components/schemas/SensorData"
+                        }
+                    }
+                },
+                "description": "Data model for the data response"
+            },
+            "GPS": {
```

Comment:
How about using a compact geojson-like representation: [long, lat (, optional altitude)]?
also, explicitely state the coordinate reference system: WGS84